cob_obstacle_distance/include/cob_obstacle_distance/distance_manager.hpp

```diff
@@ -99,6 +107,11 @@ class DistanceManager
 
         ~DistanceManager();
 
+        inline const std::string getRootFrame() const
+        {
+            return this->root_frame_;
+        }
```

Comment:
I guess in this context, "root_frame" should rather be named "frame_id" as "frame_id" in ROS is generally the parent or reference frame to which an object/mesh pose is referred...

"root_frame" in contrary (as in cob_frame_tracker) rather describes the root frame of the tf tree, i.e. the global/world frame